next.config.js

```diff
@@ -16,6 +16,16 @@ const ContentSecurityPolicy = `
   worker-src 'self' blob:;
 `;
 
+function redirectPosts(postNameMap) {
```

Comment:
Hate to do this to you but make redirectSessionTokenPosts. I like being verbose 😂